src/Microsoft.VisualStudio.Web.CodeGenerators.Mvc/RazorPage/RazorPageGenerator.cs

```diff
@@ -51,16 +51,18 @@ public class RazorPageGenerator : CommonGeneratorBase, ICodeGenerator
 
             if (string.IsNullOrEmpty(razorPageGeneratorModel.ViewName))
             {
-                // when this happens, we want to generate all the views
+                throw new ArgumentException(MessageStrings.ViewNameRequired);
```

Comment:
Looks like this change is compatible with the `2.0.0` packages, so it should be fine. 
Just need to move the null check for `ViewName` to after the `TemplateName` check and then 
:shipit: